examples/postprocessing/postprocessor_train.yaml

```diff
@@ -0,0 +1,124 @@
+postfix: "postprocessing"
+
+seed: 42
+precision: 16
+accelerator: gpu
+devices: 2
+
+dataset_root: /path/to/dataset/   # <---- specify path to your dataset
+logs_root: /path/to/logs   # <---- specify path to your logs
+dataframe_name: df.csv
+logs_folder: ${now:%Y-%m-%d_%H-%M-%S}_${postfix}
+
+num_workers: 20
+max_epochs: 1000
+valid_period: 10
+
+# CACHE EMBEDDINGS PRODUCED BY BASELINE FEATURE EXTRACTOR
+
+embeddings_cache_dir: ${dataset_root}
+
+extractor_weights: /path/to/extractor.ckpt  # <---- specify path to baseline feature extractor
+
+extractor:
+  name: vit
+  args:
+    normalise_features: True
+    use_multi_scale: False
+    weights: ${extractor_weights}
+    arch: vits16
+
+transforms_extraction:
+  name: norm_resize_hypvit_torch
+  args:
+    im_size: 224
+    crop_size: 224
+
+# TRAIN POSTPROCESSOR
+
+pairwise_model:
+  name: concat_siamese
+  args:
+    mlp_hidden_dims: [192]
+    extractor:
+      name: vit
+      args:
+        normalise_features: False
+        use_multi_scale: False
+        weights: ${extractor_weights}
+        arch: vits16
+
+hard_pairs_mining: True
+
+optimizer:
+  name: adam
```

Comment:
adamw

Reply:
oh, it's a fail
but I think it's better to keep the optimizer that we really used rather that mentioned in the paper 

and I also remember that there was no difference between adamw and adam when I trained an extractor